cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -240,7 +249,11 @@ func (c *CStorPoolController) cStorPoolDestroyEventHandler(cStorPoolGot *apis.CS
 	}
 
 	// LabelClear is to clear pool label
-	err = pool.LabelClear(cStorPoolGot.Spec.Disks.DiskList)
+	devID, err := c.getDeviceIDs(cStorPoolGot)
```

Comment:
`devID` name is confusing and seems like a single dev id.. better to change as devIDList or devIDs

Reply:
Sure

Reply:
fixed